packages/components/atoms/InfoBox/InfoBox.tsx

```diff
@@ -0,0 +1,31 @@
+import { signal } from '@preact/signals';
+import { Fragment, FunctionComponent, h } from 'preact';
+import { InfoBoxProps } from './types';
+
+const inputValue = signal('');
+
+const InfoBox: FunctionComponent<InfoBoxProps> = ({ label }: InfoBoxProps) => {
+    const handleInputChange = (event: Event) => {
+        const inputElement = event.target as HTMLInputElement;
+        inputValue.value = inputElement.value;
+    };
+
+    const handleDivClick = () => {
+        const event = new CustomEvent('buttonClick', {
+            detail: 'Event detail',
```

Comment:
check this with storybook actions, does it work for you?. BTW you need to add `bubbles: true` and `composed: true` otherwise it won't go trough the shadow dom.

Reply:
I manage to capture the events both in storybook and in test suites without adding bubbles and composed options